runtime/onert/backend/CMakeLists.txt

```diff
@@ -5,10 +5,14 @@ add_subdirectory(cl_common)
 add_subdirectory(acl_common)
 
 # Backends
+set(LIB_ONERT_BACKEND_CPU onert_backend_cpu)
```

Comment:
`training` backend has a dependency on `cpu` backend to resolve code duplication of layers.